src/main/java/org/jenkinsci/plugins/workflow/flow/FlowExecutionList.java

```diff
@@ -183,12 +174,27 @@ public boolean isResumptionComplete() {
     public static class ItemListenerImpl extends ItemListener {
         @Override
         public void onLoaded() {
-            FlowExecutionList list = FlowExecutionList.get();
```

Comment:
This part is safe to keep I think, if you have a spare minute (but fine to simply roll back everything too).

Reply:
You mean just to declare the variable and keep the comment around? Sure I can do that.

Reply:
No I meant to switch from `@Inject` to `ExtensionList.lookupSingleton` or whatever backs `.get()`.

Reply:
I went ahead and added the comment back in d923637 as well.